docs-src/security-model.md

```diff
@@ -68,32 +68,36 @@ readable by most roles, so instead we'll focus on write access for this model.
 | Application Admins | No | In Specified Namespaces | In Specified Namespaces |
 | Application Developers | No | No | In Specified Namespaces |
 
-## Namespace Restrictions
-The extra configuration options are not possible to control with RBAC. Instead,
-they will be controlled with configuration fields on GatewayClasses:
-
-* **allowedGatewayNamespaceSelector**: This field is a selector of namespaces
-  that Gateways can use this GatewayClass from. This is a standard Kubernetes
-  LabelSelector, a label query over a set of resources. The result of
-  matchLabels and matchExpressions are ANDed. Controllers must not support
-  Gateways in namespaces outside this selector. An empty selector (default)
-  indicates that Gateways can use this GatewayClass from any namespace. This
-  field is intentionally not a pointer because the nil behavior (no namespaces)
-  is undesirable here.
-
-* **allowedRouteNamespaces**: AllowedRouteNamespaces indicates in which
-  namespaces Routes can be selected for Gateways of this class. This is
-  restricted to the namespace of the Gateway by default.
-  
+## Limiting Namespaces Where a GatewayClass Can Be Used
+Kubernetes RBAC only allows for granting access to create Gateways as a whole,
+not limiting creation to specific GatewayClasses. With that in mind,
+GatewayClass includes an `allowedGatewayNamespaces` field to enable limiting
+where a GatewayClass can be used. 
+
+This field is a selector of namespaces that Gateways using this GatewayClass can
+be created from. This is a standard Kubernetes LabelSelector, a label query over
+a set of resources. Controllers must not provision new Gateways in namespaces
+outside this selector. The default empty selector indicates that Gateways can
+use this GatewayClass from any namespace.
+
+## Route Namespaces
+Service APIs allow Gateways to select Routes across multiple Namespaces.
+Although this can be remarkably powerful, this capability needs to be used
+carefully. Gateways include a `RouteNamespaces` field that allows selecting
+multiple namespaces with a label selector. By default this is limited to limit
+Gateways to select Routes within the same namespace. Additionally, Routes
```

Comment:
> By default this is limited to limit Gateways to select Routes within the same namespace.

@robscott this sentence is a tough read. Any thoughts for improvements? Maybe:
>By default, this is limited to Routes in the same namespace as the Gateway.`

Reply:
Wow, that was not great, thanks for catching that! Updated the wording to match your suggestion.

Reply:
Rob, maybe you didn't push a commit up. I'm still seeing the old wording.

Reply:
🤦 I absolutely missed pushing that fix up. I'll leave it open until you or someone else confirms I've actually resolved it this time :)